izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanel.java

```diff
@@ -98,16 +98,16 @@ public void panelActivate()
         }
         if (skip)
         {
-            logger.fine(UserPathPanel.class.getSimpleName() + " will not be shown");
+            logger.fine(UserPathPanel.NAME + " will not be shown");
```

Comment:
Static import for this constant?

Reply:
This is within the same class, renamed the variable as PANEL_NAME and using only the variable name.